templates/endpoints/DeployFunctionCalling.md

```diff
@@ -0,0 +1,42 @@
+# Serving Function calling Models
+
+You can support function calling models by using following steps.
```

Comment:
We should link to our blog and endpoints docs on function calling here to explain the mechanism